commands

```diff
@@ -73,10 +74,7 @@ case "$1" in
       dokku_log_verbose_quiet "Deleting container data"
       docker exec -it "$ID" chmod -R 777 /data
 
-      dokku_log_verbose_quiet "Stopping container"
-      docker stop "$ID" > /dev/null
-      docker kill "$ID" > /dev/null
-      sleep 1
+      service_stop "$SERVICE"
```

Comment:
You don't need to kill the container since `docker stop` will do it after a grace period (10 seconds by default). And you can't kill a stopped container, docker will output an error.